src/main/java/org/apache/mesos/hdfs/HDFSScheduler.java

```diff
@@ -236,16 +251,14 @@ public void run() {
       }
     } catch (InterruptedException | ExecutionException | InvalidProtocolBufferException e) {
       log.error("Error recovering framework id", e);
-      throw new RuntimeException(e);
     }
 
-    MesosSchedulerDriver driver = new MesosSchedulerDriver(this, frameworkInfo.build(),
-        conf.getMesosMasterUri());
+    MesosSchedulerDriver driver = new MesosSchedulerDriver(this, frameworkInfo.build(), conf.getMesosMasterUri());
     driver.run();
   }
 
   private void launchNode(SchedulerDriver driver, Offer offer,
-        String nodeName, List<String> taskNames, String executorName) {
+                          String nodeName, List<String> taskNames, String executorName) {
```

Comment:
I prefer 

I prefer: private void launchNode(SchedulerDriver driver, Offer offer,
    String nodeName, List<String> taskNames, String executorName) {
We've been trying to follow the a similar convention in HDFS. 4 space wrap and if the first parameter starts too many columns, then we make the judgement call to wrap to the next line.